index.js

```diff
@@ -80,7 +77,6 @@ export function extract (blockstore, path) {
  * @returns {AsyncIterable<Block>}
  */
 async function * exportBlocks (blockstore, block) {
-  yield block
```

Comment:
You need to yield the block _before_ the recursive call to `exportBlocks` if not yielding from within.